backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHSuite.scala

```diff
@@ -287,7 +287,7 @@ class GlutenClickHouseTPCHSuite extends GlutenClickHouseTPCHAbstractSuite {
     TestUtils.compareAnswers(result, expectedResult)
   }
 
-  test("test 'ISSUE https://github.com/Kyligence/ClickHouse/issues/225'") {
+  ignore("test 'ISSUE https://github.com/Kyligence/ClickHouse/issues/225'") {
```

Comment:
@taiyang-li please check whether this issue was fixed ?

Reply:
I can't reproduce failing in my dev environment. I will check it again later.

Reply:
@zzcclp Could we merge this PR firstly?

Reply:
> @zzcclp Could we merge this PR firstly?

OK, merged firstly, @taiyang-li @loneylee , please check the issue later.